### PR TITLE
Add salary ranges

### PR DESCRIPTION
**Pull request by ErinBecker**

### CODE REVIEW

File: pages/Deputy-Director-of-Instructor-Training-job-posting.md [hunk 1]
```diff
@@ -63,16 +63,23 @@ important part of our organisation and our strongest resource. We strive to make
...
 We’re a fiscally sponsored project of the 501(c)3 [Community Initiatives](http://communityin.org/). We have 
 [a large community](https://carpentries.org/instructors-map/) of instructors and contributors, and a much larger community of users. 
 We use evidence-based teaching practices to teach the core skills for effective computational and data work, and we promote reproducible 
 research and open science/open source widely. All interactions in The Carpentries community and spaces, both virtual and in-person, are 
 governed by our [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html#code-of-conduct-detailed-view).
 
+## Compensation and Benefits
+
+This is a remote position and is initially funded for 3 years, with the possibility of extension. For US-based candidates, compensation 
+will be $59,500 - $80,750 USD annually, commensurate with experience. Outside the US, total compensation may need to adjusted in
```
Comment (kariljordan) [inline pages/Deputy-Director-of-Instructor-Training-job-posting.md]: may need to **be** adjusted

File: pages/communications-manager.md [hunk 2]
```diff
@@ -59,16 +61,24 @@ important part of our organisation and our strongest resource. We strive to make
...
-[a large community](https://carpentries.org/instructors-map/) of instructors and contributors, and a much larger community of users.
+[a large community](https://carpentries.org/instructors-map/) of instructors and contributors, and a much larger community of users. 
 We use evidence-based teaching practices to teach the core skills for effective computational and data work, and we promote reproducible 
 research and open science/open source widely. All interactions in The Carpentries community and spaces, both virtual and in-person, are 
 governed by our [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html#code-of-conduct-detailed-view).
 
+## Compensation and Benefits
+
+This is a remote position and is initially funded for 2 years, with the possibility of extension. For US-based candidates, compensation 
+will be $61,600 - $83,600 USD annually, commensurate with experience. Outside the US, total compensation may need to adjusted in
```
Comment (kariljordan) [inline pages/communications-manager.md]: may need to **be** adjusted

File: pages/curriculum-community-developer.md [hunk 3]
```diff
@@ -62,16 +62,23 @@ important part of our organisation and our strongest resource. We strive to make
...
-use evidence-based teaching practices to teach the core skills for effective computational and data work, and we promote reproducible 
+[a large community](https://carpentries.org/instructors-map/) of instructors and contributors, and a much larger community of users. 
+We use evidence-based teaching practices to teach the core skills for effective computational and data work, and we promote reproducible 
 research and open science/open source widely. All interactions in The Carpentries community and spaces, both virtual and in-person, are 
 governed by our [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html#code-of-conduct-detailed-view).
 
+## Compensation and Benefits
+
+This is a remote position and is initially funded for 18 months, with the possibility of renewal. For US-based candidates, compensation 
+will be $77,000 - $104,500 USD annually, commensurate with experience. Outside the US, total compensation may need to adjusted in
```
Comment (kariljordan) [inline pages/curriculum-community-developer.md]: may need to *be* adjusted

File: pages/Deputy-Director-of-Instructor-Training-job-posting.md (hunk 1 above)
Comment (ErinBecker) [reply pages/Deputy-Director-of-Instructor-Training-job-posting.md]: thanks, I'll fix

File: pages/lesson-infrastructure-technology-developer.md [hunk 4]
```diff
@@ -66,20 +67,27 @@ organisation.
...
 [a large community](https://carpentries.org/instructors-map/) of instructors and contributors, and a much larger community of users. 
-We use evidence-based teaching practices to teach the core skills for effective computational and data work, and we promote reproducible
+We use evidence-based teaching practices to teach the core skills for effective computational and data work, and we promote reproducible 
 research and open science/open source widely. All interactions in The Carpentries community and spaces, both virtual and in-person, are 
 governed by our [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html#code-of-conduct-detailed-view).
 
+## Compensation and Benefits
+
+This is a remote position and is initially funded for 18 months, with the possibility of renewal. For US-based candidates, compensation
+will be $77,000 - $104,500 USD annually, commensurate with experience. Outside the US, total compensation may need to adjusted in accordance with
```
Comment (kariljordan) [inline pages/lesson-infrastructure-technology-developer.md]: may need to **be** adjusted

**Comment by kariljordan** [comment]
This is ready to merge once the corrections for adding the missing word are complete.

**Pull request merged**